DayrSuryan/DSEthiop14.xml

```diff
@@ -39,9 +39,9 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                      <title type="incomplete" ref="LIT2000Mazmur"/>
                         <textLang mainLang="gez"/>
                      <note>Contains Psalms 1-6 and 8-17; Psalm 7 is missing.
-                     Numeration of Psalms is not consequent. Nummer 1 is used two times;
-                   Psalms 5, 6, 8, 9, 11 have no numbers. Recent numbers written
-                   in pencil, Eastern Arabic numerals, are added to all Psalms.</note>
+                     The original numeration of Psalms is not consequent. Number 1 is used two times;
```

Comment:
I a "note" after the title, you need to say that it is a portion of the Psalter (only Ps 1-6 and 8-17).

Reply:
So "A portion of the Psalter (Ps 1-6 and 8-17)" instead of the current formulation?

Reply:
So "A portion of the Psalter (Ps 1-6 and 8-17)" instead of the current formulation?

Reply:
Yes, for instance.